### ISSUE TITLE
Color Options for dark / light theme

### ISSUE THREAD

**Issue by feedhopper** [code-of-chaos/obsidian-colored_tags_wrangler#50]
I use this plugin every day and I love it very much!

There is one thing, however: In order to react to the daily change of light in my surroundings, I use this plugin:

https://github.com/CyberT17/obsidian-day-and-night

It automatically switches the theme from dark to light and vice versa.

I tried for a while to find colors that match both themes but it does not actually work because the text color switches with my theme and that makes a lot of combinations I statically picked in your plugin incompatible. The result is text that barely can be read.

Is it possible to add another row of color pickers (background and text) for the two scenarios (theme switched to light / dark)? If this is too much work: I'd be happy to only have a setting for the background.

This would be GREAT!

Thank you so much for your patience. And first of all: for this really cool plugin!

**Comment by AnnaSasDev** [comment]
Yes I love this idea and (although due to some irl stuff I haven't been able to fully work on it yet) in the rewrite of the plugin, this should be easily done through an "extension" on the plugin.
Definitely something I'll start development on soon, though I don't have a set timeline yet.

**Comment by feedhopper** [comment]
That sounds great! Thank you very much!!

**Comment by AnnaSasDev** [comment]
added to to-do-list of `1.0.0` rework -> #52
